presto-main/src/main/resources/webapp/src/components/PageTitle.jsx

```diff
@@ -27,6 +29,29 @@ type State = {
     errorText: ?string,
 }
 
+function ClusterResourceGroupNavBar({titles, urls, current} : Props) {
```

Comment:
In react, we always need to use a[ unique key](https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key) when rendering a list of elements, and preferably it should not be the index.

```
function ClusterResourceGroupNavBar({titles, urls, current = 0}: Props) {

    const navBarItems = titles.map((title, index) => {
        const isActive = current === index;
        const className = isActive ? "navbar-brand" : "navbar-brand inactive";

        return (
            <td key={index}>
                <span className={className}>
                    {isActive || !urls?.length ? title : <a href={urls[index]}>{title}</a>}
                </span>
            </td>
        );
    });

    return <>{navBarItems}</>;
}
```